frontend/app/services/api-notifications-service.js

```diff
@@ -26,37 +26,26 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-// TODO move to UI components
-module.exports = function($rootScope, $timeout, ConfigurationService) {
+module.exports = function(NotificationsService, ApiHelper) {
+  'use strict';
 
-  return {
-    restrict: 'E',
-    replace: true,
-    scope: {},
-    templateUrl: '/templates/components/flash_message.html',
-    link: function(scope, element, attrs) {
-      $rootScope.$on('flashMessage', function(event, message) {
-        scope.message = message;
-        scope.flashType = 'notice';
-        scope.flashId = 'flash-notice';
+  var addError = function(error) {
+    var messages = ApiHelper.getErrorMessages(error);
 
-        var fadeOutTime = attrs.fadeOutTime || 3000;
+    if (messages.length > 1) {
+      NotificationsService.addError('', messages);
+    }
+    else {
+      NotificationsService.addError(messages[0], []);
```

Comment:
@ulferts This sounds reasonable. My main concern was really the repetitive use of the empty second parameter. :+1: